src/Store.js

```diff
@@ -90,12 +104,25 @@ export default class Store {
     return undefined;
   }
 
+  _createAndFetchFrom(values, source) {
+    const item = new this._Item({ store: this, autoSave: this._options.autoSave });
+    item.construct(values, { source });
+    item.fetch(source).then(() => {
+      this.items.push(item);
+      this.incompleteItems = this.incompleteItems.filter(incompleteItem => incompleteItem !== item);
+    });
+    this.incompleteItems.push(item);
+    return item;
+  }
+
   _createItems(rawItems, source) {
     return Promise.all(rawItems.map((rawItem) => {
       const item = new this._Item({ store: this, autoSave: this._options.autoSave });
-      if (rawItem._transporterState !== STATE.BEING_DELETED &&
-        rawItem._transporterState !== STATE.DELETED &&
-        rawItem._transporterState !== STATE.LOCKED) {
+      if (rawItem._transporterState === STATE.BEING_DELETED ||
+        rawItem._transporterState === STATE.DELETED ||
+        rawItem._transporterState === STATE.LOCKED) {
+        this.incompleteItems.push(item);
```

Comment:
Do you want to leave these items in the incompleteItems array? Because I can't see where they are removed from there.